wallet/src/de/schildbach/wallet/ui/send/PaymentProtocolFragment.kt

```diff
@@ -236,40 +182,38 @@ class PaymentProtocolFragment : Fragment(R.layout.fragment_payment_protocol) {
                 }
             }
         }
+    }
 
-        paymentProtocolModel.onSendCoinsOffline.observe(viewLifecycleOwner) { (status, data) ->
```

Comment:
The pattern of observing the result of a send is replaced with suspend calls which is simpler and more straightforward.